Polyhedron/demo/Polyhedron/Scene_plane_item.h

```diff
@@ -133,11 +133,15 @@ public Q_SLOTS:
   }
   
   void setNormal(float x, float y, float z) {
-    frame->setOrientation(x, y, z, 0.f);
+    QVector3D normal(x,y,z);
+    QVector3D origin(0,0,1);
+    QQuaternion q(CGAL::sqrt((normal.lengthSquared()) * (origin.lengthSquared())) + QVector3D::dotProduct(origin, normal),QVector3D::crossProduct(origin, normal));
```

Comment:
I found that on stack overflow : http://stackoverflow.com/questions/1171849/finding-quaternion-representing-the-rotation-from-one-vector-to-another